src/main/scala/overflowdb/codegen/CodeGen.scala

```diff
@@ -31,63 +24,59 @@ class CodeGen(schemaFile: String, basePackage: String) {
   def writeConstants(outputDir: JFile): JFile = {
     val baseDir = File(outputDir.getPath + "/" + basePackage.replaceAll("\\.", "/")).createDirectories
 
-    def writeConstantsFile(className: String, constants: List[Constant])(mkSrc: Constant => String): Unit = {
-      val src = constants.map { constant =>
-        val documentation = constant.comment.filter(_.nonEmpty).map(comment => s"""/** $comment */""").getOrElse("")
-        s""" $documentation
-           | ${mkSrc(constant)}
-           |""".stripMargin
+    def writeConstantsFile[A](className: String, constants: Seq[A])(makeDocAndSource: A => (Option[String], String)): Unit = {
```

Comment:
Maybe a type alias or explicite type for `makeDocAndSource` with some doc for enhances readability?

Reply:
done in adfe00f